src/components/organisms/Sider/Sider.tsx

```diff
@@ -101,10 +108,53 @@ const Sider: React.FC = () => {
     {
       icon: 'cog',
       onClick: () => navigate('/settings'),
+      title: 'Settings',
+    },
+    {
+      dropdownComponent: (
+        <Popover
+          placement="rightBottom"
+          content={
+            <>
+              <DropdownListItem>
+                <a href={externalLinks.documentation} target="_blank">
+                  <Text color={Colors.slate300} className="regular middle">
+                    Documentation
+                  </Text>
+                </a>
+              </DropdownListItem>
+              <DropdownListItem>
+                <a href={externalLinks.discord} target="_blank">
+                  <Text color={Colors.slate300} className="regular middle">
+                    Discord community
+                  </Text>
+                </a>
+              </DropdownListItem>
+              <DropdownListItem>
+                <a href={externalLinks.github} target="_blank">
+                  <Text color={Colors.slate300} className="regular middle">
+                    GitHub
+                  </Text>
+                </a>
+              </DropdownListItem>
+            </>
+          }
+          trigger={['hover']}
+        >
+          <QuestionCircleOutlined
+            style={{
+              fontSize: 20,
+            }}
+          />
```

Comment:
Optionally, as it's pretty short:
```suggestion
          <QuestionCircleOutlined style={{  fontSize: 20 }} />
```